infrastructure/helm/keycloak/keycloak-setup.md

```diff
@@ -0,0 +1,64 @@
+# Install Quantumserverless and Configure Keycloak
+## In the command line
+Switch default namespace to the install target namespace
+```
+kubectl config set-context –current –namespace="target namespace"
+```
+Install helm chart
+```
+helm install "release name" quantemserverless
+```
+Wati for all pods come up except `releasename-kuberay-head-xxxx`
+Run ./login-setup.sh <RELEASE NAME>
+
+## In the browser
+# Configure Keycloak
+
+Open browser at http://<LOCALIP>:<KEYCLOAK PORT>
+ - Select Administration Console
+ - Login with “user” / <PASSWORD>
+  <LOCALIP>, <KEYCLOAK PORT>, <PASSWORD> are in the login-setup.sh script output.
+
+Create “quantumserverless” Realm
```

Comment:
What do you think about changing it to: https://www.keycloak.org/operator/realm-import

Reply:
This import capability is provided the keycloak operator which is not installed by the helm chart.  The operator provides another way to install software on the cluster.  We install the keycloak using helm chart so we probably don't want to install the operator.  WDYT?

Reply:
There are other import options like: https://www.keycloak.org/server/importExport#_importing_a_realm_during_startup

I don't know if that one could work 🤔

Reply:
It worked. Thanks!  I put he exported realm into a configmap and mounted it to the keycloak container.
```
  extraVolumes:
    - name: realm
      configMap:
        name: keycloakrealm
  extraVolumeMounts:
    - name: realm
      mountPath: /opt/bitnami/keycloak/data/import
  extraStartupArgs: "--import-realm"
```